### PR TITLE
chore: AutoSuggestSample Uno 4.1.9

### PR DESCRIPTION
**Pull request by iury-kc**
AutoSuggestSample updated to  Uno 4.1.9

### CODE REVIEW

File: UI/AutoSuggestSample/AutoSuggestSample/AutoSuggestSample.Wasm/LinkerConfig.xml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ﻿<linker>
 	<assembly fullname="Uno.UI">
-		<type fullname="*" />
+	<assembly fullname="Microsoft.Extensions.Options" />
+	<type fullname="*" />
```
Comment (jeromelaban) [inline UI/AutoSuggestSample/AutoSuggestSample/AutoSuggestSample.Wasm/LinkerConfig.xml:4]: This is not valid.

Comment (iury-kc) [reply UI/AutoSuggestSample/AutoSuggestSample/AutoSuggestSample.Wasm/LinkerConfig.xml:4]: Agree. It was there before, and wasn't sure if would be required somewhere.  Removing.

File: UI/AutoSuggestSample/AutoSuggestSample/AutoSuggestSample.Wasm/LinkerConfig.xml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ﻿<linker>
 	<assembly fullname="Uno.UI">
-		<type fullname="*" />
+	<assembly fullname="Microsoft.Extensions.Options" /> 
 	</assembly>
```
Comment (jeromelaban) [inline UI/AutoSuggestSample/AutoSuggestSample/AutoSuggestSample.Wasm/LinkerConfig.xml:4]:
```suggestion
```

File: UI/AutoSuggestSample/AutoSuggestSample/AutoSuggestSample.Wasm/LinkerConfig.xml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ﻿<linker>
 	<assembly fullname="Uno.UI">
```
Comment (jeromelaban) [inline UI/AutoSuggestSample/AutoSuggestSample/AutoSuggestSample.Wasm/LinkerConfig.xml:2]:
```suggestion
	<assembly fullname="Uno.UI" />
```

**Review by jeromelaban — APPROVED** [review]

**Review by jeromelaban — APPROVED** [review]

**Pull request merged**